### PR TITLE
Pre-card updates

### PR DESCRIPTION
**Pull request by pyrello**
The purpose of this PR is to extract some updates from #6054 that are not directly related to the card updates to hopefully make the process of reviewing and testing that PR a little easier.

A lot of these changes are minor syntactical changes handled automatically by PhpStorm.

# How to test
```
ddev blt ds --site sandbox.uiowa.edu && ddev drush @sandbox.local uli
```
- Review all code changes. 
- Pay attention to the code changes from double quotes and single quotes to ensure that is the only thing that changed. If any of them seem concerning, test the corresponding functionality.
- Horizontal menu `accessible-menu` library updates with no regressions. Check Tippie?
- Layout Builder styles patch to prevent duplicate classes from rendering. Find example of this on Sandbox and confirm that duplicate classes are no longer showing, but that classes are still showing.
- Article list block still defaults to 10 items per page.
- Blocks that show a placeholder in Layout Builder still have the `layout-builder-block--placeholder` added to them.
- When changing the media format style for card and image blocks, the underlying view mode changes.
- No regressions for narrow article feature.
- `inner--grid` class still gets removed from blocks where `inner--list` is selected.
- Headline and button classes still get set correctly on relevant blocks.
- Ensure that the card block button indicator functionality still renders correctly.

## Admissions
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli
```
- `admissions_core` module weight has changed. Spot check functionality to ensure there are no regressions.
- Compare AoS fields to ensure that they look the same: Link, Scholarship.
- Compare event series fields to ensure that they look the same: Button link, Summary.
- Compare major fields to ensure they look the same: CC checkpoints, Checkpoints.

## Now
```
ddev blt ds --site now.uiowa.edu && ddev drush @now.local uli
```
- `now_core` module weight has changed. Spot check functionality to ensure there are no regressions.

## Pharmacy
```
ddev blt ds --site pharmacy.uiowa.edu && ddev drush @pharmacy.local uli
```
- Compare person pages between the prod site and local to ensure no regressions.

## Grad Thesis Defense
```
ddev blt ds --site grad.uiowa.edu && ddev drush @grad.local uli
```
- Check thesis defense fields to ensure they look the same: Date, Location.

### CODE REVIEW

File: config/default/core.entity_view_display.block_content.uiowa_card.default.yml [hunk 1]
```diff
@@ -76,4 +68,5 @@ hidden:
   entity_print_view_epub: true
   entity_print_view_pdf: true
   entity_print_view_word_docx: true
+  field_uiowa_card_button_display: true
```
Comment (joewhitsitt) [inline config/default/core.entity_view_display.block_content.uiowa_card.default.yml:71]: was this intentional? just checking

Comment (pyrello) [reply config/default/core.entity_view_display.block_content.uiowa_card.default.yml:71]: Yeah. I think that is field that just determines the whether the button displays and we don't want to actually show it.

**Comment by joewhitsitt** [comment]
Looking good except for the icon set/field template bits.

**Comment by pyrello** [comment]
We discovered that the updates for Grad Thesis Defense field icon updates wasn't working. They are dependent on some updates that weren't previously included, including the base field template. Since it would be really nice to clear all this out anyways, I decided to just pull everything field-related that doesn't appear to be dependent on the rest of the changes in #6054.

@bspeare I could potentially use some help tomorrow crafting some test instructions and/or maybe we could do so more paired testing.

**Review by joewhitsitt — APPROVED** [review]
spot checked the recent changes across a couple sites. yesterday's testing looked pretty good as well. TIL about item_tag after noticing it for the first time. nothing noticeable to me, but looks like percy noticed a small position placement of the icon.

**Comment by pyrello** [comment]
Thanks for testing this, @joewhitsitt!

**Pull request merged**